js/ui/appletManager.js

```diff
@@ -598,6 +601,23 @@ function createApplet(extension, appletDefinition, panel = null) {
     return applet;
 }
 
+function _addAppletStyleClassUuid(applet, metadata){
+    const appletClass = _generateAppletStyleClassUuid(metadata)
+    if(appletClass !== ''){
+        applet._addStyleClass(appletClass);
+        return true
+    }
+    return false
+}
+
+function _generateAppletStyleClassUuid(metadata){
+    if(typeof metadata === 'object' && metadata !== null && typeof metadata.uuid === 'string'){
```

Comment:
Could this be `if(metadata && typeof metadata.uuid === 'string')` instead?  Or if it metadata could be a string at some point then `if(metadata && typeof metadata === 'object' && typeof metadata.uuid === 'string')` ?

Reply:
No problems. Just noticed that line and wondered if all those checks were necessary, which is sounds like they are.